src/component/NavbarCustom.js

```diff
@@ -1,35 +1,64 @@
 import React from "react";
-import { Container, Navbar, Nav, NavDropdown } from "react-bootstrap";
+import { Container, Navbar, Nav } from "react-bootstrap";
+
+const navigationItems = {
+    Cabinet: ["Presentation", "Equipe", "Missions"],
+    Formation: ["Catalogue", "Sur mesure"]
+};
 
 class NavbarCustom extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            hasSubMenu: false,
+            activeSubMenu: "",
+        };
+        this.setHasSubMenu = this.setHasSubMenu.bind(this);
 
+    }
+    setHasSubMenu(newActiveSubMenu) {
```

Comment:
Ici tu ne set pas seulement le flag hasSubMenu, tu set aussi activeSubMenu.
Perso, je pense que ce serait mieux d'appeler la fonction "setSubMenu".

Reply:
True that